src/twisted/words/test/test_irc.py

```diff
@@ -9,13 +9,27 @@
 import operator
 import time
 
+from twisted.internet import protocol, task
```

Comment:
@glyph Where do you want `from __future__ import unicode_literals` to be added?
I ran an experiment and added it to:

```
twisted/words/im/ircsupport.py
twisted/words/protocols/irc.py
twisted/words/test/test_irc.py
twisted/words/test/test_irc_service.py
twisted/words/test/test_ircsupport.py
```
and some of the tests started breaking on Python 2, without breaking on Python 3.